### ISSUE TITLE
dns.promises.setServers should be ignored from prefer-promises/dns

### ISSUE THREAD

**Issue by titanism** [mysticatea/eslint-plugin-node#348]
Right now you have to either use `dns.promises.setServers([ ... ])` or use the ignore line.

`dns.promises.setServers` does not actually return a Promise, and will actually error if you call it with `.then()`.

```js
// eslint-disable-next-line n/prefer-promises/dns
dns.setServers([
  '...'
]);
```

**Comment by voxpelli** [comment]
Try [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n), it is the maintained version of this module.

We switched to it in eg. [`eslint-config-standard`](https://github.com/standard/eslint-config-standard) / [`standard`](https://github.com/standard/standard) and it is maintained by me and other members of the official [ESLint community](https://github.com/eslint-community) organization.

If it isn't fixed there, try creating an issue there as well.
